src/events/components/Attendance/AttendButton.tsx

```diff
@@ -55,6 +55,20 @@ const AttendButton: FC<IAttendButtonProps> = (props: IAttendButtonProps) => {
       toggleModal={toggleModal}
       setCaptcha={signUp}
       errorText={FAILED_CAPTCHA_ERROR_TEXT}
+      onError={async (error) => {
+        const user = await getUser();
+        Sentry.captureEvent({
+          message: 'User failed turnstile challenge when signing up for event.',
```

Comment:
Vi burde ha en egen ordbok akkurat for event-domenet med norsk og engelsk "melde seg på" = register eller sign up?

Reply:
Eeenig!